src/libs/workspace-utils.ts

```diff
@@ -183,9 +184,10 @@ export const groupConstraintMessage = (workspace: WorkspaceWrapper): string | un
     _.uniq,
     _.sortBy(_.identity)
   )(groupConstraintPolicies);
-  return `Data Access Controls add additional permission restrictions to a workspace. These were added when you imported data from a controlled access source. All workspace collaborators must also be members of the following groups: ${requiredGroups.join(
-    ', '
-  )}.`;
+  return `Data Access Controls add additional permission restrictions to a workspace. These were added when you imported data from a controlled access source. All workspace collaborators must also be members of the following ${pluralize(
+    'groups',
```

Comment:
Does this work? I've always passed in the singular for it to pluralize.

Reply:
Turns out it does:
```
> const pluralize = (await import('pluralize')).default;
undefined
> pluralize('groups', 1)
'group'
> pluralize('groups', 3)
'groups'
```

But this was not intentional. Changed to match convention in https://github.com/DataBiosphere/terra-ui/pull/4469/commits/e65a25037b01478f64dd37b5fea8bcc92e19e9ee.